howtolens/chapter_1_introduction/scripts/tutorial_8_fitting.py

```diff
@@ -396,7 +396,7 @@
 
 fit = al.FitImaging(masked_imaging=masked_imaging, tracer=tracer)
 
-aplt.FitImaging.subplot_fit_imaging(fit=fit, include=aplt.Include(mask=True))
+aplt.FitImaging.subplot_fit_imaging(fit=fit, include=aplt.Include2D(mask=True))

```

Comment:
This include argument is interesting. What's the motivation for making the value an instance of a class rather than just using a flag?

Reply:
Primarily API - its one object a user can be familiar with using for all visualization.

I toyed with making flags (e.g. include_mask, include_critical_curves, etc.) in every visualization class (or method) but it also gets very nasty in terms of number of parameters.

The Include also uses config files to laod the 'default' included stuff on plots if a user doesn't input anything.